tests/cases/hash_state_test.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2016 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+from .. import base
+from girder.utility import hash_state
+import hashlib
+import this
+
+def setUpModule():
+    base.startServer()
+
+
+def tearDownModule():
+    base.stopServer()
+
+
+# Define a few variables used by all the tests:
+# Use the python's zen as 'long' string
+test_chunk = this.s.encode('utf8')
+# Arbitrary number to be able to split the python zen in a least a few chunks
+chunk_size = len(test_chunk) / 6
+chunks = [test_chunk[i:i+chunk_size] for i in range(0, len(test_chunk), chunk_size)]
+
+class HashStateTestCase(base.TestCase):
+
+
+    def _simpleHashingTest(self, hash_type):
+        """
+        Test the hashing of hash_state against python's hashlib
+
+        :param hash_type: String representing the hash algorithm to use
+        """
+        official_hasher = hashlib.new(hash_type)
```

Comment:
Girder's Python code uses camelCase for variable names. Let's change the underscore_names, to ensure consistent style.